src/components/NewMessageModal.vue

```diff
@@ -99,6 +102,7 @@ export default {
 				accountId: data.account,
 				subject: data.subject,
 				body: data.isHtml ? data.body.value : toPlain(data.body).value,
+				editorBody: data.body.value,
```

Comment:
unused

Reply:
this one is needed, because action "updateMessage" needs editorBody. If i would remove this, it would be impossible to open again the undo-sendet-message